### PR TITLE
HBASE-24500 The behavior of RegionInfoBuilder.newBuilder(RegionInfo) …

### PR DESCRIPTION
**Pull request by Apache9**
…is strange

### CODE REVIEW

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 29s |  Docker mode activated.  |
| -0 :warning: |  yetus  |   0m  3s |  Unprocessed flag(s): --brief-report-file --spotbugs-strict-precheck --whitespace-eol-ignore-list --whitespace-tabs-ignore-list --quick-hadoopcheck  |
||| _ Prechecks _ |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 23s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   3m 35s |  master passed  |
| +1 :green_heart: |  compile  |   2m  6s |  master passed  |
| +1 :green_heart: |  shadedjars  |   5m 32s |  branch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   1m 13s |  master passed  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 16s |  Maven dependency ordering for patch  |
| -1 :x: |  mvninstall  |   1m 45s |  root in the patch failed.  |
| -1 :x: |  compile  |   0m 37s |  hbase-server in the patch failed.  |
| -0 :warning: |  javac  |   0m 37s |  hbase-server in the patch failed.  |
| -1 :x: |  shadedjars  |   4m  7s |  patch has 16 errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   1m 12s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   0m 44s |  hbase-protocol-shaded in the patch passed.  |
| +1 :green_heart: |  unit  |   1m  4s |  hbase-client in the patch passed.  |
| -1 :x: |  unit  |   0m 38s |  hbase-server in the patch failed.  |
|  |   |  26m 23s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.11 Server=19.03.11 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1850/1/artifact/yetus-jdk8-hadoop3-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/1850 |
| Optional Tests | javac javadoc unit shadedjars compile |
| uname | Linux 096938a592c1 4.15.0-58-generic #64-Ubuntu SMP Tue Aug 6 11:12:41 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / 08c97d3a87 |
| Default Java | 1.8.0_232 |
| mvninstall | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1850/1/artifact/yetus-jdk8-hadoop3-check/output/patch-mvninstall-root.txt |
| compile | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1850/1/artifact/yetus-jdk8-hadoop3-check/output/patch-compile-hbase-server.txt |
| javac | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1850/1/artifact/yetus-jdk8-hadoop3-check/output/patch-compile-hbase-server.txt |
| shadedjars | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1850/1/artifact/yetus-jdk8-hadoop3-check/output/patch-shadedjars.txt |
| unit | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1850/1/artifact/yetus-jdk8-hadoop3-check/output/patch-unit-hbase-server.txt |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1850/1/testReport/ |
| Max. process+thread count | 340 (vs. ulimit of 12500) |
| modules | C: hbase-protocol-shaded hbase-client hbase-server U: . |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1850/1/console |
| versions | git=2.17.1 maven=(cecedd343002696d0abb50b32b541b8a6ba2883f) |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 32s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +0 :ok: |  prototool  |   0m  0s |  prototool was not available.  |
| +1 :green_heart: |  hbaseanti  |   0m  0s |  Patch does not have any anti-patterns.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 23s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   3m 33s |  master passed  |
| +1 :green_heart: |  checkstyle  |   1m 52s |  master passed  |
| +1 :green_heart: |  spotbugs  |   6m  3s |  master passed  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 14s |  Maven dependency ordering for patch  |
| -1 :x: |  mvninstall  |   1m 42s |  root in the patch failed.  |
| +1 :green_heart: |  checkstyle  |   0m  9s |  The patch passed checkstyle in hbase-protocol-shaded  |
| +1 :green_heart: |  checkstyle  |   0m 30s |  hbase-client: The patch generated 0 new + 189 unchanged - 4 fixed = 189 total (was 193)  |
| +1 :green_heart: |  checkstyle  |   1m  0s |  hbase-server: The patch generated 0 new + 0 unchanged - 1 fixed = 0 total (was 1)  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| -1 :x: |  hadoopcheck  |   1m 51s |  The patch causes 16 errors with Hadoop v3.1.2.  |
| -1 :x: |  hadoopcheck  |   3m 48s |  The patch causes 16 errors with Hadoop v3.2.1.  |
| -1 :x: |  hbaseprotoc  |   0m 33s |  hbase-server in the patch failed.  |
| -1 :x: |  spotbugs  |   0m 30s |  hbase-server in the patch failed.  |
||| _ Other Tests _ |
| +1 :green_heart: |  asflicense  |   0m 32s |  The patch does not generate ASF License warnings.  |
|  |   |  29m 12s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.11 Server=19.03.11 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1850/1/artifact/yetus-general-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/1850 |
| Optional Tests | dupname asflicense spotbugs hadoopcheck hbaseanti checkstyle cc hbaseprotoc prototool |
| uname | Linux e96a2a80f353 4.15.0-60-generic #67-Ubuntu SMP Thu Aug 22 16:55:30 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / 08c97d3a87 |
| mvninstall | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1850/1/artifact/yetus-general-check/output/patch-mvninstall-root.txt |
| hadoopcheck | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1850/1/artifact/yetus-general-check/output/patch-javac-3.1.2.txt |
| hadoopcheck | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1850/1/artifact/yetus-general-check/output/patch-javac-3.2.1.txt |
| hbaseprotoc | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1850/1/artifact/yetus-general-check/output/patch-hbaseprotoc-hbase-server.txt |
| spotbugs | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1850/1/artifact/yetus-general-check/output/patch-spotbugs-hbase-server.txt |
| Max. process+thread count | 94 (vs. ulimit of 12500) |
| modules | C: hbase-protocol-shaded hbase-client hbase-server U: . |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1850/1/console |
| versions | git=2.17.1 maven=(cecedd343002696d0abb50b32b541b8a6ba2883f) spotbugs=3.1.12 |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   1m 40s |  Docker mode activated.  |
| -0 :warning: |  yetus  |   0m  3s |  Unprocessed flag(s): --brief-report-file --spotbugs-strict-precheck --whitespace-eol-ignore-list --whitespace-tabs-ignore-list --quick-hadoopcheck  |
||| _ Prechecks _ |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 28s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   5m 12s |  master passed  |
| +1 :green_heart: |  compile  |   3m  4s |  master passed  |
| +1 :green_heart: |  shadedjars  |   6m 58s |  branch has no errors when building our shaded downstream artifacts.  |
| -0 :warning: |  javadoc  |   0m 27s |  hbase-client in master failed.  |
| -0 :warning: |  javadoc  |   0m 44s |  hbase-server in master failed.  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 15s |  Maven dependency ordering for patch  |
| -1 :x: |  mvninstall  |   2m 39s |  root in the patch failed.  |
| -1 :x: |  compile  |   0m 54s |  hbase-server in the patch failed.  |
| -0 :warning: |  javac  |   0m 54s |  hbase-server in the patch failed.  |
| -1 :x: |  shadedjars  |   5m 20s |  patch has 16 errors when building our shaded downstream artifacts.  |
| -0 :warning: |  javadoc  |   0m 30s |  hbase-client in the patch failed.  |
| -0 :warning: |  javadoc  |   0m 45s |  hbase-server in the patch failed.  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   1m  8s |  hbase-protocol-shaded in the patch passed.  |
| +1 :green_heart: |  unit  |   1m 29s |  hbase-client in the patch passed.  |
| -1 :x: |  unit  |   0m 50s |  hbase-server in the patch failed.  |
|  |   |  35m 58s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.11 Server=19.03.11 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1850/1/artifact/yetus-jdk11-hadoop3-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/1850 |
| Optional Tests | javac javadoc unit shadedjars compile |
| uname | Linux 71fac8e0c290 4.15.0-91-generic #92-Ubuntu SMP Fri Feb 28 11:09:48 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / 08c97d3a87 |
| Default Java | 2020-01-14 |
| javadoc | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1850/1/artifact/yetus-jdk11-hadoop3-check/output/branch-javadoc-hbase-client.txt |
| javadoc | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1850/1/artifact/yetus-jdk11-hadoop3-check/output/branch-javadoc-hbase-server.txt |
| mvninstall | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1850/1/artifact/yetus-jdk11-hadoop3-check/output/patch-mvninstall-root.txt |
| compile | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1850/1/artifact/yetus-jdk11-hadoop3-check/output/patch-compile-hbase-server.txt |
| javac | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1850/1/artifact/yetus-jdk11-hadoop3-check/output/patch-compile-hbase-server.txt |
| shadedjars | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1850/1/artifact/yetus-jdk11-hadoop3-check/output/patch-shadedjars.txt |
| javadoc | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1850/1/artifact/yetus-jdk11-hadoop3-check/output/patch-javadoc-hbase-client.txt |
| javadoc | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1850/1/artifact/yetus-jdk11-hadoop3-check/output/patch-javadoc-hbase-server.txt |
| unit | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1850/1/artifact/yetus-jdk11-hadoop3-check/output/patch-unit-hbase-server.txt |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1850/1/testReport/ |
| Max. process+thread count | 208 (vs. ulimit of 12500) |
| modules | C: hbase-protocol-shaded hbase-client hbase-server U: . |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1850/1/console |
| versions | git=2.17.1 maven=(cecedd343002696d0abb50b32b541b8a6ba2883f) |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

File: hbase-protocol-shaded/src/main/protobuf/HBase.proto [hunk 1]
```diff
@@ -79,7 +79,6 @@ message RegionInfo {
   optional bool offline = 5;
   optional bool split = 6;
   optional int32 replica_id = 7 [default = 0];
-  optional string region_encoded_name = 8;
```
Comment (infraio) [inline hbase-protocol-shaded/src/main/protobuf/HBase.proto:82]: This is not included in branch-2.2. So this is never released?

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   1m 42s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +0 :ok: |  prototool  |   0m  0s |  prototool was not available.  |
| +1 :green_heart: |  hbaseanti  |   0m  0s |  Patch does not have any anti-patterns.  |
| +1 :green_heart: |  @author  |   0m  1s |  The patch does not contain any @author tags.  |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 20s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   4m  6s |  master passed  |
| +1 :green_heart: |  checkstyle  |   1m 57s |  master passed  |
| +1 :green_heart: |  spotbugs  |   6m 46s |  master passed  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 14s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   3m 58s |  the patch passed  |
| +1 :green_heart: |  checkstyle  |   0m  8s |  The patch passed checkstyle in hbase-protocol-shaded  |
| +1 :green_heart: |  checkstyle  |   0m 33s |  hbase-client: The patch generated 0 new + 192 unchanged - 4 fixed = 192 total (was 196)  |
| +1 :green_heart: |  checkstyle  |   1m 13s |  hbase-server: The patch generated 0 new + 4 unchanged - 1 fixed = 4 total (was 5)  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  hadoopcheck  |  12m 47s |  Patch does not cause any errors with Hadoop 3.1.2 3.2.1.  |
| +1 :green_heart: |  hbaseprotoc  |   2m 21s |  the patch passed  |
| +1 :green_heart: |  spotbugs  |   8m 13s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  asflicense  |   0m 37s |  The patch does not generate ASF License warnings.  |
|  |   |  54m 21s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.11 Server=19.03.11 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1850/2/artifact/yetus-general-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/1850 |
| Optional Tests | dupname asflicense spotbugs hadoopcheck hbaseanti checkstyle cc hbaseprotoc prototool |
| uname | Linux d83a3782ccd4 4.15.0-74-generic #84-Ubuntu SMP Thu Dec 19 08:06:28 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / 61a7468df7 |
| Max. process+thread count | 84 (vs. ulimit of 12500) |
| modules | C: hbase-protocol-shaded hbase-client hbase-server U: . |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1850/2/console |
| versions | git=2.17.1 maven=(cecedd343002696d0abb50b32b541b8a6ba2883f) spotbugs=3.1.12 |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   2m 12s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +1 :green_heart: |  hbaseanti  |   0m  0s |  Patch does not have any anti-patterns.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 24s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   4m 19s |  master passed  |
| +1 :green_heart: |  checkstyle  |   1m 53s |  master passed  |
| +1 :green_heart: |  spotbugs  |   2m 58s |  master passed  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 14s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   3m 26s |  the patch passed  |
| +1 :green_heart: |  checkstyle  |   0m 32s |  hbase-client: The patch generated 0 new + 189 unchanged - 4 fixed = 189 total (was 193)  |
| +1 :green_heart: |  checkstyle  |   1m  7s |  hbase-server: The patch generated 0 new + 0 unchanged - 1 fixed = 0 total (was 1)  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  hadoopcheck  |  11m 24s |  Patch does not cause any errors with Hadoop 3.1.2 3.2.1.  |
| +1 :green_heart: |  spotbugs  |   3m 15s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  asflicense  |   0m 26s |  The patch does not generate ASF License warnings.  |
|  |   |  39m 51s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.11 Server=19.03.11 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1850/3/artifact/yetus-general-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/1850 |
| Optional Tests | dupname asflicense spotbugs hadoopcheck hbaseanti checkstyle |
| uname | Linux be0579bcf86c 4.15.0-60-generic #67-Ubuntu SMP Thu Aug 22 16:55:30 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / b2ec4c1ea0 |
| Max. process+thread count | 94 (vs. ulimit of 12500) |
| modules | C: hbase-client hbase-server U: . |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1850/3/console |
| versions | git=2.17.1 maven=(cecedd343002696d0abb50b32b541b8a6ba2883f) spotbugs=3.1.12 |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

File: hbase-protocol-shaded/src/main/protobuf/HBase.proto (hunk 1 above)
Comment (Apache9) [reply hbase-protocol-shaded/src/main/protobuf/HBase.proto]: Yes, so we should remove it before the 2.3.0 release.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   2m  6s |  Docker mode activated.  |
| -0 :warning: |  yetus  |   0m  3s |  Unprocessed flag(s): --brief-report-file --spotbugs-strict-precheck --whitespace-eol-ignore-list --whitespace-tabs-ignore-list --quick-hadoopcheck  |
||| _ Prechecks _ |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 44s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   5m 49s |  master passed  |
| +1 :green_heart: |  compile  |   2m  8s |  master passed  |
| +1 :green_heart: |  shadedjars  |   7m 39s |  branch has no errors when building our shaded downstream artifacts.  |
| -0 :warning: |  javadoc  |   0m 32s |  hbase-client in master failed.  |
| -0 :warning: |  javadoc  |   0m 47s |  hbase-server in master failed.  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 15s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   5m 28s |  the patch passed  |
| +1 :green_heart: |  compile  |   2m  6s |  the patch passed  |
| +1 :green_heart: |  javac  |   2m  6s |  the patch passed  |
| +1 :green_heart: |  shadedjars  |   7m 42s |  patch has no errors when building our shaded downstream artifacts.  |
| -0 :warning: |  javadoc  |   0m 35s |  hbase-client in the patch failed.  |
| -0 :warning: |  javadoc  |   0m 54s |  hbase-server in the patch failed.  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   1m 51s |  hbase-client in the patch passed.  |
| +1 :green_heart: |  unit  | 193m 24s |  hbase-server in the patch passed.  |
|  |   | 234m  0s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.11 Server=19.03.11 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1850/3/artifact/yetus-jdk11-hadoop3-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/1850 |
| Optional Tests | javac javadoc unit shadedjars compile |
| uname | Linux 3362cf04bdc2 4.15.0-74-generic #84-Ubuntu SMP Thu Dec 19 08:06:28 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / b2ec4c1ea0 |
| Default Java | 2020-01-14 |
| javadoc | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1850/3/artifact/yetus-jdk11-hadoop3-check/output/branch-javadoc-hbase-client.txt |
| javadoc | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1850/3/artifact/yetus-jdk11-hadoop3-check/output/branch-javadoc-hbase-server.txt |
| javadoc | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1850/3/artifact/yetus-jdk11-hadoop3-check/output/patch-javadoc-hbase-client.txt |
| javadoc | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1850/3/artifact/yetus-jdk11-hadoop3-check/output/patch-javadoc-hbase-server.txt |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1850/3/testReport/ |
| Max. process+thread count | 3707 (vs. ulimit of 12500) |
| modules | C: hbase-client hbase-server U: . |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1850/3/console |
| versions | git=2.17.1 maven=(cecedd343002696d0abb50b32b541b8a6ba2883f) |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   1m  7s |  Docker mode activated.  |
| -0 :warning: |  yetus  |   0m  4s |  Unprocessed flag(s): --brief-report-file --spotbugs-strict-precheck --whitespace-eol-ignore-list --whitespace-tabs-ignore-list --quick-hadoopcheck  |
||| _ Prechecks _ |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 22s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   3m 55s |  master passed  |
| +1 :green_heart: |  compile  |   1m 21s |  master passed  |
| +1 :green_heart: |  shadedjars  |   6m  4s |  branch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   0m 58s |  master passed  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 14s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   3m 43s |  the patch passed  |
| +1 :green_heart: |  compile  |   1m 21s |  the patch passed  |
| +1 :green_heart: |  javac  |   1m 21s |  the patch passed  |
| +1 :green_heart: |  shadedjars  |   6m  5s |  patch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   0m 57s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   1m  6s |  hbase-client in the patch passed.  |
| +1 :green_heart: |  unit  | 205m 13s |  hbase-server in the patch passed.  |
|  |   | 234m 42s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.11 Server=19.03.11 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1850/3/artifact/yetus-jdk8-hadoop3-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/1850 |
| Optional Tests | javac javadoc unit shadedjars compile |
| uname | Linux f4096ed218b1 4.15.0-101-generic #102-Ubuntu SMP Mon May 11 10:07:26 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / b2ec4c1ea0 |
| Default Java | 1.8.0_232 |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1850/3/testReport/ |
| Max. process+thread count | 3030 (vs. ulimit of 12500) |
| modules | C: hbase-client hbase-server U: . |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1850/3/console |
| versions | git=2.17.1 maven=(cecedd343002696d0abb50b32b541b8a6ba2883f) |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

**Pull request merged**